### ISSUE TITLE
500 error

### ISSUE THREAD

**Issue by dadodasyra** [s0ftik3/reverso-api#25]
I guess it's the reverso official api to blame but I'm reporting it anyway, I'm getting this on every requests
```
Error: Request failed with status code 500
     at #request (/root/Histbot/node_modules/reverso-api/src/reverso.js:535:19)
     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
     at async Reverso.getSpellCheck (/root/Histbot/node_modules/reverso-api/src/reverso.js:191:22)
```

**Comment by dadodasyra** [comment]
When using SpellCheck I mean, I didn't tried for translations or other things like this

**Comment by dadodasyra** [comment]
When trying the link in a browser to reproduce the get request it returns:
For https://orthographe.reverso.net/api/v1/Spelling?text=Test&language=fra&getCorrectionDetails=true
```json
{"message":"Object reference not set to an instance of an object.","errorType":"internal"}
```

**Comment by dadodasyra** [comment]
The GET api is probably not maintained by reverso as the official website uses post requests

**Comment by dadodasyra** [comment]
Any news ? Am I the only one using this ?

**Comment by s0ftik3** [comment]
fixed in the patch version 2.22.4, please update the package
